PalasoUIWindowsForms/ClearShare/LicenseInfo.cs

```diff
@@ -69,13 +71,46 @@ public virtual bool EditingAllowed
 		/// what is ethical. We expect that the vast majority of people are going to abide by them.
 		/// </summary>
 		public string RightsStatement {get; set; }
+
+		protected virtual string GetBestLicenseTranslation(string idSuffix, string englishText, string comment,
+			IEnumerable<string> languagePriorityIds, out string idOfLanguageUsed)
+		{
+			idSuffix = "MetadataDisplay.Licenses."+idSuffix;
+			foreach (var targetLanguage in languagePriorityIds)
+			{
+				if (targetLanguage == "en")
+				{
+					//do the query to make sure the string is there to be translated someday
+					var unused = LocalizationManager.GetDynamicString("Palaso", idSuffix, englishText, comment);
```

Comment:
Is L10NSharp going to do something bad if we ask it for an English translation?

I can imagine someone using the localization mechanism (don't know whether it would work, admittedly) to correct typos in the built-in strings, or adapt them to basic English.

I could also imagine someone using this library in a program basically created in French or Russian, where the fall-back-if-there-is-no-localization actually isn't English.

YAGNI, I guess...but I don't see how it helps to have the special case for English, unless L10NSharp does something silly when asked for English.